### ISSUE TITLE
Update date facets to reflect recent ML change (from 2007)

### ISSUE THREAD

**Issue by gigamorph** [project-lux/lux-frontend#76]
NOTE:  this should remain open as there will be a solution on the backend for estimates and once that is completed, this work can be reimplemented.  Blocked by https://github.com/project-lux/lux-marklogic/issues/47

-------------------------------
**Problem Description:**
Date facets are failing on the results page.

**Expected Behavior/Solution:**
The date facets requests should be updated to reflect the recent change in ML date formatting.

**Requirements:**

1. The request made to the backend should be structured in the same manner as the advanced search, which is outlined [here](https://git.yale.edu/lux-its/lux-web/issues/1878#issue-23063).
2. The selected facets should parse the new format.

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [ ] ~Wireframe/Mockup - Heather~
- [ ] ~Committee discussions - Sarah~
- [ ] ~Feasibility/Team discussion - Sarah~
- [ ] ~Backend requirements - TBD~
- [x] Frontend requirements- Kameryn
- [ ] ~Questions~

**UAT/LUX Examples:**

Old example that now fails:
https://lux-front-tst.collections.yale.edu/view/results/works?q=%7B%22AND%22%3A%5B%7B%22text%22%3A%22William%22%2C%22_lang%22%3A%22en%22%7D%2C%7B%22text%22%3A%22Blake%22%2C%22_lang%22%3A%22en%22%7D%5D%7D&sq=William+Blake&wp=1&wf=%7B%22AND%22%3A%5B%7B%22createdDate%22%3A%221700%22%2C%22_comp%22%3A%22%3E%3D%22%7D%2C%7B%22createdDate%22%3A%221800%22%2C%22_comp%22%3A%22%3C%3D%22%7D%5D%7D&facetRequest=true

New example that passes:
https://lux-front-tst.collections.yale.edu/view/results/works/?q=%7B%22AND%22%3A%5B%7B%22text%22%3A%22William%22%2C%22_lang%22%3A%22en%22%7D%2C%7B%22text%22%3A%22Blake%22%2C%22_lang%22%3A%22en%22%7D%5D%7D&sq=William+Blake&wf=%7B%22AND%22%3A%5B%7B%22createdDate%22%3A%7B%22start%22%3A%221700%22%2C%22end%22%3A%222000%22%7D%7D%5D%7D&facetRequest=true&wp=1

**Dependencies/Blocks:**
Blocked by - https://github.com/project-lux/lux-marklogic/issues/47

**Related Github Issues:**

- #1878 

**Related links:**
N/A

**Wireframe/Mockup:**
N/A

**Comment by clarkepeterf** [comment]
This ticket is no longer needed. It is based on a ML change that no longer exists. (Passing in `start` and `end` to dates, rather than using the `_comp` operators)

**Comment by roamye** [comment]
Approved by UAT 9/12 to close.
